cmd/byohctl/utils/logger.go

```diff
@@ -0,0 +1,240 @@
+package utils
```

Comment:
Thinking about this some more - why are we defining our own logger? Why not use zap?